docs/docs-content/clusters/cluster-management/os-patching.md

```diff
@@ -16,71 +16,170 @@ upstream repositories as improvements, bug fixes, and security patches are relea
+   this option, your node will not be able to reboot even if a patch requires it and the respective patch may not
+   succeed.
+
+4. Finish cluster creation and deploy your cluster.
 
 To enable **OS Patching Schedule**, ensure you are in the **Settings** step of the cluster creation wizard. Next, click
 on the **Manage machines** tab, and select the drop-down input for **OS Patching Schedule**.
 
-To enable **OS Patching Schedule**, for an active cluster. Navigate to the **Main Menu** and click on **Clusters**. In
-the cluster view page, find the row for the respective cluster you want to configure OS patching for. Click on the three
-dots at the end of row to access the cluster settings. Next, click on the **Machine Management** tab, and select the
-drop-down input for **OS Patching Schedule**.
+</TabItem>
+
+</Tabs>
+
+### Validate
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+2. Click on the cluster you patched to view cluster details.
+
+3. Click on the **Nodes** tab of cluster details page. You can find the current OS version and the time the most
+   recently patch was applied.
+
+   | **Field**               | **Description**                         |
+   | ----------------------- | --------------------------------------- |
+   | Last Applied Patch Time | The date and time of the last OS patch. |
+   |                         |                                         |
+   | Patched Version         | The latest patched version.             |
+
+## Perform On-Demand Patching
+
+You can request an OS patch on-demand after you have created your cluster. This option provides a way for you to perform
+an immediate update.
+
+### Limitations
+
+- This operation is not available for existing Kubernetes clusters imported into Palette.
+- This operation is not available or work as expected for managed Kubernetes Services such as EKS and AKS.
+  - For EKS clusters, an OS update can be triggered from Palette. This would initiate a request on AWS to update cluster
+    node groups to the latest patch version.
+
+### Prerequisite
 
-# On-Demand
+- An active cluster in Palette.
 
-This option provides a way for you to perform an immediate update.
+### Instructions
 
-To perform an **On-Demand** update for an active cluster. Navigate to the **Main Menu** and click on **Clusters**. In
-the cluster view page, find the row for the respective cluster you want to configure OS patching for. Click on the three
-dots at the end of row to access the cluster settings. Next, click on the **Machine Management** tab, and select the
-drop-down input for **OS Patching Schedule**.
+1. Log in to [Palette](https://console.spectrocloud.com).
 
-<br />
+2. From the left **Main Menu**, click **Cluster**. Select the cluster you want to patch to view cluster details.
 
-:::warning
+3. In the upper-right corner, click on **Settings** and select **On-Demand Update**. Doing so will trigger an OS update
+   immediately if a newer version is available.
```

Comment:
```suggestion
3. In the upper-right corner, click on **Settings** and select **On-Demand Update**. This will trigger an OS update
   immediately if a newer version is available.
```